File: Service/Implement/DanhMucATTPTinhTrangService.cs
namespace Service.Implement
{
    public class DanhMucATTPTinhTrangService : BaseService<DanhMucATTPTinhTrang, IDanhMucATTPTinhTrangRepository>
    , IDanhMucATTPTinhTrangService
    {
    private readonly IDanhMucATTPTinhTrangRepository _DanhMucATTPTinhTrangRepository;
    public DanhMucATTPTinhTrangService(IDanhMucATTPTinhTrangRepository DanhMucATTPTinhTrangRepository) : base(DanhMucATTPTinhTrangRepository)
    {
    _DanhMucATTPTinhTrangRepository = DanhMucATTPTinhTrangRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucATTPTinhTrang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucATTPTinhTrang } from 'src/app/shared/DanhMucATTPTinhTrang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucATTPTinhTrangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucATTPTinhTrang";
    }
}

